src/app/src/test/kotlin/org/imperial/mrc/hint/integration/ProjectTests.kt

```diff
@@ -31,8 +28,7 @@ class ProjectTests : VersionFileTests() {
     private val testState = "{\"state\": \"test\"}"
 
     @Test
-    fun `can create new project`()
-    {
+    fun `can create new project`() {
```

Comment:
Huh, all these methods have been changed to have opening { on same line as signature - I thought your preference was to put the opening { on a new line?

Reply:
As we don't have shared project style settings for hint, my IDE is using the kotlin defaults, which is brace on same line as here. We did discuss moving to using the more space-y style we use in montagu/OW, but that hasn't been implemented yet. I don't know why your IDE has formatted things with braces on next line unless you have that set as your default for kotlin?

Reply:
(nb I'm going to work on the style settings as part of "maintenance afternoon" this afternoon!)